apps/auth/config/index.ts

```diff
@@ -14,7 +14,9 @@ export const jwtSecret = process.env.JWT_SECRET || 'abcddddd';
 export const refreshTokenSecret = process.env.REFRESH_TOKEN_SECRET || 'a secret 2 phrase!!';
 export const tokenExpiration = process.env.TOKEN_EXPIRATION || '10m';
 export const refreshTokenExpiration = process.env.REFRESH_TOKEN_EXPIRATION || '30d';
-export const cookieTokenVerificationTime = Number(process.env.COOKIE_TOKEN_VERIFICATION_TIME || TEN_MINUTES);
+export const cookieTokenVerificationTime = Number(process.env.COOKIE_TOKEN_VERIFICATION_TIME || 1);//TEN_MINUTES
+export const processedCookiexpiration =Number(process.env.PROCESSED_COOKIE_EXPIRATION || 30);
```

Comment:
what is this variable mean?
also, `Expiration` should be with capital E.

Reply:
I wanted to use it , instead of hard- coded , to delete the token from cache after 30 seconds

Reply:
ok great. just fix the typo of the non-capital E. :)